README.md

```diff
@@ -11,4 +10,141 @@ UAVCAN is an open lightweight protocol designed for reliable communication in ae
 robust vehicle bus networks.
 
 * [**UAVCAN website**](http://uavcan.org)
-* [**UAVCAN mailing list**](https://groups.google.com/forum/#!forum/uavcan)
+* [**UAVCAN forum**](https://forum.uavcan.org)
+
+## Identifier ranges
+
+### Subjects
+
+For message subjects, the following range mapping is adopted (limits inclusive).
+Unused ranges are reserved for future expansion of adjacent ranges.
+
+From    | To        | Purpose
+--------|-----------|------------------------------------------------
+0       | 32767     | Dynamic
+61440   | 62803     | Vendor-specific public static
+62804   | 65535     | Standard static
+
+### Services
+
+For services, the following range mapping is adopted (limits inclusive).
+Unused ranges are reserved for future expansion of adjacent ranges.
+
+From    | To        | Purpose
+--------|-----------|------------------------------------------------
+0       | 127       | Dynamic
+256     | 383       | Vendor-specific public static
+384     | 511       | Standard static
+
+## Standard static identifier allocation
+
+### Subjects
+
+Ordered by priority from high to low.
+
+Namespace                   | Lower bound (inclusive)
+----------------------------|-------------------------
+`uavcan.time`               | 62804
+`uavcan.node`               | 62805
+`uavcan.pnp`                | 62810/65533
+`uavcan.internet`           | 65510
+`uavcan.diagnostic`         | 65520
+
+### Services
+
+Ordered by priority from high to low.
+
+Namespace                   | Lower bound (inclusive)
+----------------------------|-------------------------
+`uavcan.register`           | 384
+`uavcan.pnp`                | 390
+`uavcan.file`               | 400
+`uavcan.node`               | 430
+`uavcan.internet`           | 500
+
+## Generic data type definitions
+
+### SI
+
+The namespace `uavcan.si` contains a collection of generic data types describing commonly used
+physical quantities.
+
+Some of the messages are time stamped, in which case the time stamp is always situated at the end in order
+to facilitate binary compatibility with non-timestamped messages (both time-stamped and non-time-stamped
+messages have identical headers, so if the time stamp is not required they can be used interchangeably).
+Names of the time stamped messages have the suffux `TS`.
+
+All units follow the [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units).
+All units are unscaled basic units of measure (e.g., meters rather than kilometers, kilograms rather than milligrams),
+unless a different multiplier is explicitly specified in the definition (e.g., nanosecond).
+
+All coordinate systems are right-handed.
+In relation to body, the preferred standard is as follows: **X** -- forward, **Y** -- right, **Z** -- down.
```

Comment:
We use a different coordinate frame: NED instead of ENU, because the formed is more widespread in aerospace. This was discussed at length years ago (on Skype, no records are available anymore), can't change easily.

Reply:
Okay. I like aerospace.